pages/login.vue

```diff
@@ -92,7 +70,6 @@ const submitForm = (event: Event) => {
       rememberToken: rememberToken.value,
```

Comment:
tham khảo 
https://jasonwatmore.com/post/2022/07/25/vue-3-pinia-user-registration-and-login-example-tutorial#auth-store-js